src/pray.c

```diff
@@ -763,7 +757,12 @@ int godnum;
 				newsym(mtmp->mx, mtmp->my);
 			}
 		}
-		u.ugoatblesscnt = rnz(300);
```

Comment:
The tabbing here seems confusing. Maybe the github view is just weird, or maybe there's an old bug here.